activejob/test/cases/argument_serialization_test.rb

```diff
@@ -208,6 +209,8 @@ def assert_arguments_roundtrip(args)
     end
 
     def perform_round_trip(args)
-      ActiveJob::Arguments.deserialize(ActiveJob::Arguments.serialize(args))
+      ArgumentsRoundTripJob.perform_later(*args) # Actually performed inline
+
+      JobBuffer.last_value
```

Comment:
This `JobBuffer` doesn't seem to get cleared in between tests, which feels weird, so I may open another PR to add clearing it as a `teardown` step.

I noticed because I originally wanted to assert about the entire contents of `JobBuffer`, but it contained unrelated stuff, which is why I went with `last_value` instead.